### PR TITLE
Add `sortBy` to `sort` autofixer for `no-array-prototype-extensions` rule

### PR DESCRIPTION
**Pull request by tgvrssanthosh**
## Summary
Added auto-fixer for `no-array-prototype-extensions` which replaces `sortBy` with `sort`.

## Description
Recently, the proposal to deprecate array prototype extensions has been approved and got merged. The plan is to add autoFixers to replace ember array prototype extensions with the native array methods. In this PR, an auto fixer has been added which will replace the ember array method `sortBy` with the native array method `sort`.

## Testing
Modified test case to check if the right output is generated after the fix.

### CODE REVIEW

**Comment by tgvrssanthosh** [comment]
@bmish Can you review this PR as well?

**Comment by tgvrssanthosh** [comment]
> @bmish Can you review this PR as well?

@bmish Just found a small issue. Will resolve it and will ping you.

File: tests/lib/rules/no-array-prototype-extensions.js [hunk 1]
```diff
@@ -565,10 +565,120 @@ import { get as g } from 'dummy';
...
     {
+      // When unexpected number of arguments are passed, auto-fixer will not run
       code: 'something.sortBy()',
       output: null,
       errors: [{ messageId: 'main', type: 'CallExpression' }],
     },
+    {
+      // Arguments containing strings
+      code: "something.sortBy('abc', 'def')",
+      output: `import { compare } from '@ember/utils';
```
Comment (bmish) [inline tests/lib/rules/no-array-prototype-extensions.js:576]: Can we make this a lot more concise? I don't think we need one-time-use variables, comments, or newlines.

Here's what I would output:

```js
something.sort((a, b) => {
  for (const key of ['abc', 'def']) {
    const compareValue = compare(get(a, key), get(b, key));
    if (compareValue) {
      return compareValue;
    }
  }
  return 0;
});
```

Comment (tgvrssanthosh) [reply tests/lib/rules/no-array-prototype-extensions.js:576]: Makes sense. I just copied over the existing implementation of the `sortBy` function. Will make it concise as you suggested.

**Comment by tgvrssanthosh** [comment]
@bmish Addressed the comment. Can you take a relook?

**Review by bmish — APPROVED** [review]
Thanks!

**Pull request merged**